main/reference/zoe-api/zcfseat.md

```diff
@@ -1,9 +1,11 @@
 # ZCFSeat Object
 
 Zoe uses **seats** to access or manipulate offers. Seats represent active
-offers and let contracts and users interact with them. Zoe has two kinds
-of seats. **ZCFSeats** are used within contracts and with **zcf** methods.
-**UserSeats** represent offers external to Zoe and the contract.
+offers and let contracts and users interact with them. Two kinds of seats
+represent a single position, and allow interaction from within a contract
```

Comment:
The clause ", and allow interaction from within a contract and from the outside." should be deleted. It's redundant with the following sentences and is unclear.

Reply:
done